### PR TITLE
Gamma

### LINKED ISSUE #117: Gamma correction
Implement gamma correction along with brig. and cont.

### PR DESCRIPTION
**Pull request by henpe071**
Added the option of changing the gamma value in the manipulator widget. The gamma functions the same way as contrast and brightness does.

Closes #117

### CODE REVIEW

File: ViAn/GUI/videowidget.cpp [hunk 1]
```diff
@@ -1504,22 +1508,16 @@ void VideoWidget::zoom_label_finished() {
     zoom_label->clearFocus();
 }
 
-int VideoWidget::get_brightness() {
```
Comment (henpe071) [inline ViAn/GUI/videowidget.cpp:1512]: These were unused

**Review by niklasniklas — CHANGES_REQUESTED** [review]
No gamma in report. (reportgenerator.cpp line 61)

**Comment by niklasniklas** [comment]
Otherwise fine! :-)

**Review by niklasniklas — APPROVED** [review]
Works fine

**Pull request merged**